src/__tests__/cli.ts

```diff
@@ -44,8 +44,6 @@ async function exec(
 
     process.chdir(testProjectDir);
 
-    const main = require('..').main as (args: CliArguments) => Promise<void>;
```

Comment:
keeping the import inside the `exec` function was important when we were trying to just call the index file and have it work, but the way the async ran gave us nothing to hook into to know when execution was complete so we opted to exporting `main` and calling it directly instead, making dynamically referencing the code less useful.